src/Compilers/Core/Portable/PortableShim.cs

```diff
@@ -413,6 +413,11 @@ internal static class Encoding
                 .GetTypeInfo()
                 .GetDeclaredMethod(nameof(GetEncoding), paramTypes: new[] { typeof(int) })
                 .CreateDelegate<Func<int, System.Text.Encoding>>();
+
+            internal static readonly Func<System.Text.Encoding> Default = Type
+                .GetTypeInfo()
+                .GetDeclaredMethod("get_" + nameof(Default), paramTypes: new Type[] { })
```

Comment:
You're getting a member of another class by using the `nameof` a member of the current class. I think that doesn't really make sense, since they have no compile-time association. Please spell this "get_Default".